test/trustedresources.go

```diff
@@ -116,7 +116,8 @@ func SetupTrustedResourceConfig(ctx context.Context, verificationNoMatchPolicy s
 // This function helps to setup 3 kinds of VerificationPolicies:
 // 1. One public key in inline data
 // 2. One public key in secret
-// 3. 2 authorities referring to the same secret. This is to test and make sure we don't have duplicate counts
+// 3. wrong inline key
```

Comment:
I don't understand the reason for this comment change

Reply:
this comment was wrong, so I updated it reflect the current case